src/projects/yaq/src/index.html

```diff
@@ -0,0 +1,156 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Quiz App</title>
+    <link rel="stylesheet" href="public/styles/styles.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <div class="app">
+      <header>
+        <div class="greeting">
+          <div class="good-morning">
+            <img src="public/svg/sun.svg" alt="" />
+            <span> GOOD MORNING</span>
+          </div>
+          <h1>Madelyn Dias</h1>
+        </div>
+        <div class="profile-pic">
+          <img src="public/svg/avatar.svg" alt="" />
+        </div>
+      </header>
+
+      <div class="recent-quiz">
+        <div class="quiz-info">
+          <span class="quiz-label">RECENT QUIZ</span>
```

Comment:
Instead of bolding these letters on HTML, we can use `strong` or `em` to emphasize them. Or, if the design requires it to be all caps, we can use CSS to make that design.

Reply:
sure I will look into it